src/AddTodoForm.js

```diff
@@ -1,10 +1,22 @@
 import { React } from "react";
 
-const AddTodoForm = () => {
+const AddTodoForm = (props) => {
+    function handleAddTodo(event) {
+        event.preventDefault();
+        const todoTitle = event.target.title.value;
+        console.log(todoTitle);
```

Comment:
You can remove the console.log statement since it is primarily intended for debugging purposes.